apolloschurchapp/src/tabs/discover/__snapshots__/discover.tests.js.snap

```diff
@@ -437,14 +437,104 @@ exports[`The Discover tab component Should retrieve the Content Channel Feeds 1`
       </View>
     </View>
     <RCTScrollView
+      ListEmptyComponent={
+        <Err
+          error={
+            [Error: Network error: No more mocked responses for the query: query getFeatureFeed($featureFeedId: ID!) {
```

Comment:
Looks like this query is still erroring, but there's more data here than before.